src/solvers/alternating_update/region_update.jl

```diff
@@ -112,6 +112,7 @@ function region_update(
     outputlevel,
     info...,
     region_kwargs...,
+    inserter_kwargs...,
```

Comment:
An alternative design would be to pass `inserter_kwargs` into `all_kwargs` as a NamedTuple instead of splatting them. That would keep those keyword arguments more self contained, i.e. make it clearer which keyword arguments are meant for the inserter. Do you think that would work with the rest of this PR?

Reply:
I do think that change keeps things more organized & it was easy to implement.
I just updated this PR with that change.

Reply:
I'm confused, I still see `inserter_kwargs` being splatted on this line, did you mean to update that?

Reply:
Yes, I did mean to update it. Actually it was a lucky mistake because I just noticed the code worked anyway. So I did some more digging and `region_kwargs` which is already being splatted into `all_kwargs` already contains `inserter_kwargs`. So the real solution (apart from possibly larger changes) is to just remove that line about `inserter_kwargs` altogether since they are already, subtly, being passed.